engineyard/ey-cookbooks-stable-v5
# Adding custom yarn recipe

#### Description of your patch
Adding the ability to install any `yarn` version available on https://github.com/yarnpkg/yarn/releases/

#### Recommended Release Notes
Adding the ability to install any `yarn` version available on https://github.com/yarnpkg/yarn/releases/

#### Estimated risk
Low

#### Components involved
This is a new custom recipe

#### Description of testing done
See QA instructions

#### QA Instructions
- Boot QA stack
- Enable `yarn` recipe
- Execute `yarn --version` on any instance verifying that it's being installed
